tests/ert_tests/ensemble_evaluator/prefect/conftest.py

```diff
@@ -399,16 +399,19 @@ def mock_ws_monitor(unused_tcp_port):
 @pytest.fixture()
 def external_sum_function(tmpdir):
     with tmpdir.as_cwd():
-        # Create temporary module that defines a function `bar`
-        # 'bar' returns a call to different function 'internal_call' defined in the same python file
+        # Create temporary module that defines a function `bar` 'bar' returns a
```

Comment:
` 'bar' 'bar' ` reads weird to me, maybe change to ` 'bar', which ` or similar?

Reply:
thanks, this is `vim` that has reflowed the text. Will fix.